annotations/annotations_api.go

```diff
@@ -48,6 +48,7 @@ func (api *annotationsAPI) Get(ctx context.Context, contentUUID string) (*http.R
 		return nil, err
 	}
 
+	userAgent(apiReq)
```

Comment:
I am pretty sure that this is not related to this PR but I am wondering if this function should be moved in a specific package since it is something that can be reused by several app components... or maybe something that can be "automagically" added by the http client?... If it is too complicated leave its as it is... 😅